docs/examples/space-destroyer.md

```diff
@@ -42,6 +42,7 @@ ship.setFlag(SpriteFlag.StayInScreen, true)
 ship.bottom = 120
 controller.moveSprite(ship, 100, 100)
 info.setLife(3)
+effects.starField.startScreenEffect()
```

Comment:
How long does a screen effect last by default?

Reply:
Forever, and it takes precedence over other effects so it shouldn't ever turn off